plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -623,4 +642,89 @@ private static void validateOrcBloomFilterColumns(ConnectorTableMetadata tableMe
             throw new TrinoException(INVALID_TABLE_PROPERTY, format("Orc bloom filter columns %s not present in schema", Sets.difference(ImmutableSet.copyOf(orcBloomFilterColumns), allColumns)));
         }
     }
+
+    public static Optional<String> getLatestMetadataLocation(TrinoFileSystem fileSystem, String location)
```

Comment:
Typically we put helper methods below the calling code, so I'd put this below `getMetadataLocation`

Reply:
This can also be private

Reply:
Rearranged the methods. 
I am using `getLatestMetadataLocation` in `TestIcebergRegisterTableProcedure` as well.